docs/docs/coverage/os/conda.md

```diff
@@ -16,21 +16,24 @@ Trivy detects packages that have been installed with `Conda`.
 ### `<package>.json`
 Trivy parses `<conda-root>/envs/<env>/conda-meta/<package>.json` files to find the version and license for the dependencies installed in your env.
 
-### `environment.yml`[^2]
-Trivy supports parsing [environment.yml][environment.yml][^2] files to find dependency list.
+### `environment.yml`[^1]
+Trivy supports parsing [environment.yml][environment.yml][^1] files to find dependency list.
 
-!!! note
-    License detection is currently not supported.
-
-`environment.yml`[^2] files supports [version range][env-version-range]. We can't be sure about versions for these dependencies.
-Therefore, you need to use `conda env export` command to get dependency list in `Conda` default format before scanning `environment.yml`[^2] file.
+`environment.yml`[^1] files supports [version range][env-version-range]. We can't be sure about versions for these dependencies.
+Therefore, you need to use `conda env export` command to get dependency list in `Conda` default format before scanning `environment.yml`[^1] file.
 
 !!! note
     For dependencies in a non-Conda format, Trivy doesn't include a version of them.
 
+#### licenses
```

Comment:
It is complicated because the licence is included in the SBOM, but I think it should be a separate section rather than under SBOM.

```
## SBOM
### `<package>.json`
### `environment.yml`[^1]

## License
### `<package>.json`
### `environment.yml`[^1]
```

Or

```
## `<package>.json`
### SBOM
### License

## `environment.yml`[^1]
### SBOM
### License
```

Reply:
updated - https://github.com/aquasecurity/trivy/pull/6953/commits/26e339128d4f9880556b7084e72a62ad048f8d5f